atlassian/bitbucket/cloud/repositories/users.py

```diff
@@ -0,0 +1,75 @@
+from requests.models import ProtocolError
+from ..base import BitbucketCloudBase
+from datetime import datetime
+
+
+class User(BitbucketCloudBase):
```

Comment:
I would move the `User` to  `atlassian/bitbucket/cloud/common/user.py` and use this as base for all other useers. E.G. The `delete` is implemented in `DefaultReviewer`, in the inherited class also other properties can be added.

Reply:
Ok will do this. `Participant` will be a part of `pullRequests.py`

Reply:
I alos suggest to remove the `delete` method from the `User`.